spec.html

```diff
@@ -34508,10 +34508,10 @@ <h2>Syntax</h2>
           [+UnicodeMode] `u{` CodePoint `}`
 
         UnicodeLeadSurrogate ::
-          &gt; any Unicode code point in the inclusive range 0xD800 to 0xDBFF
+          &gt; any Unicode code point with a value in the inclusive range 0xD800 to 0xDBFF
```

Comment:
```suggestion
          &gt; any Unicode code point in the inclusive range U+D800 to U+DBFF
```
(and likewise the next)

Reply:
Sure, accepted.